contrib/net/http/http_test.go

```diff
@@ -416,3 +416,27 @@ func handler200(w http.ResponseWriter, _ *http.Request) {
 func handler500(w http.ResponseWriter, _ *http.Request) {
 	http.Error(w, "500!", http.StatusInternalServerError)
 }
+
+func BenchmarkHttpServeTrace(b *testing.B) {
```

Comment:
:+1: 

Thanks for adding the benchmark. Should we add this to the benchmark CI job?

Reply:
Yes, but only after it's in `main` I believe